lib/membrane_live_audio_mixer.ex

```diff
@@ -88,11 +92,20 @@ defmodule Membrane.LiveAudioMixer do
         |> Map.put(:live_queue, nil)
         |> Map.put(:stream_format, nil)
         |> Map.put(:end_of_stream?, false)
+        |> Map.put(:status, :init)
```

Comment:
I'd favor converting this into boolean `started?` as there are only two possible values